examples/sst2/input_pipeline.py

```diff
@@ -0,0 +1,175 @@
+# Copyright 2020 The Flax Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""SST-2 input pipeline."""
+
+# pylint: disable=too-many-arguments,import-error,too-many-instance-attributes,too-many-locals
+import collections
+from typing import Dict, Sequence, Text
+
+from absl import logging
+
+import numpy as np
+import tensorflow.compat.v2 as tf
+import tensorflow_datasets as tfds
+
+def build_vocab(datasets: Sequence[tf.data.Dataset],
+                special_tokens: Sequence[Text] = (b'<pad>', b'<unk>', b'<s>', b'</s>'),
```

Comment:
I was wondering why to use bytes and not python strings?

Reply:
TFDS is returning text fields as bytes so I sticked to that for performance. Sounds like a good idea to make this consistent where possible.